libs/order/src/models/order-payment.ts

```diff
@@ -0,0 +1,16 @@
+import { DaffOrder } from './order';
+
+export interface DaffOrderCreditCardPayment {
+  payment_id: number;
```

Comment:
number | string -- we should probably make an `id` type.

Reply:
changed